src/NATS.Client/Api/Duration.cs

```diff
@@ -0,0 +1,52 @@
+﻿namespace NATS.Client.Api
+{
+    public class Duration
```

Comment:
```suggestion
    public sealed class Duration
```

What's our target for .NET? This should likely be a readonly struct.

Reply:
fixing